test/e2e/storage/volume_provisioning.go

```diff
@@ -543,12 +540,7 @@ var _ = SIGDescribe("Dynamic Provisioning", func() {
 			class := newStorageClass(test, ns, "external")
 			className := class.Name
 			claim := newClaim(test, ns, "external")
-			// the external provisioner understands Beta only right now, see
-			// https://github.com/kubernetes-incubator/external-storage/issues/37
```

Comment:
This should have been fixed in https://github.com/kubernetes-incubator/external-storage/pull/38.
Ping @wongma7 for confirmation.

Reply:
yes, is fine to remove this comment. thanks!